docs/COMPONENTS.md

```diff
@@ -21,57 +24,61 @@ Components use the following definition:
         ...
     ],
 
-    // How this component is selectable.
-    // By "default" the user can click inside the component to select it.
-    // The "handle" option adds a draggable bar with the component name at the right top of the component.
-    // This option is useful for components that have non clickable content (such as iframes).
+    // Method of moving a component to a different position:
```

Comment:
It's more about how you can select components. Primarily to see their properties but indeed also to move them. Iframes is given as example but maybe containers should be added as example too